crates/framework/Cargo.toml

```diff
@@ -7,11 +7,9 @@ homepage = "https://github.com/hulks/hulk"
 
 [dependencies]
 parking_lot = { workspace = true }
-structs = { workspace = true }
 types = { workspace = true }
 
 [build-dependencies]
```

Comment:
Remove `[build-dependencies]` because there is no `build.rs` anymore?

Reply:
Done